test/externalTests/gnosis.sh

```diff
@@ -87,6 +87,10 @@ function gnosis_safe_test
     sed -i "s|it\(('can only be called from Safe itself'\)|it.skip\1|g" test/libraries/Migration.spec.ts
     sed -i "s|it\(('should enforce delegatecall to MultiSend'\)|it.skip\1|g" test/libraries/MultiSend.spec.ts
 
+    jq '.overrides."@ethersproject/abstract-provider"="5.6.0" |
+        .overrides."@ethersproject/abstract-signer@5.6.0"
+                  ."@ethersproject/abstract-provider"="5.6.0"' package.json > package.json.tmp && mv package.json.tmp package.json
```

Comment:
We're running with `set -e` so you don't really need the `&&` here. You can put it on a separate line.
```suggestion
    jq '.overrides."@ethersproject/abstract-provider"="5.6.0" |
        .overrides."@ethersproject/abstract-signer@5.6.0"."@ethersproject/abstract-provider"="5.6.0"' package.json > package.json.tmp
    mv package.json.tmp package.json
```
```